content/python/concepts/sets/terms/issuperset/issuperset.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: '.issuperset()'
+Description: 'Checks whether all elements in specified set exist in the original set.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Collections'
+  - 'Methods'
+  - 'Sets'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.issuperset()`** method returns `True` if all the items in the specified set exists in the original set,otherwise it returns `False`.
```

Comment:
```suggestion
In Python, the **`.issuperset()`** method is used to determine if the original set contains all elements of the specified set. It returns `True` if every element of the specified set is also an element of the original set, and `False` otherwise. 
```